cpu/atmega_common/periph/rtt.c

```diff
@@ -249,11 +392,16 @@ void rtt_set_alarm(uint32_t alarm, rtt_cb_t cb, void *arg)
     else {
         DEBUG("RTT alarm interrupt not active\n");
     }
+#endif
```

Comment:
`rtt_set_alarm()` only has 4 lines of shared code. It would be cleaner to just have two versions of the function and one `#ifdef`.